src/test/scala/com/microsoft/hyperspace/index/rules/JoinIndexRuleTest.scala

```diff
@@ -426,7 +431,7 @@ class JoinIndexRuleTest extends HyperspaceRuleTestSuite with SQLHelper {
   }
 
   /** Returns tuple of left and right base relation paths for a logical plan */
```

Comment:
Not related to this PR, but can you update this comment since this doesn't return a "tuple"?

Reply:
thanks @imback82 , fixed